lib/impress.jstp.js

```diff
@@ -89,11 +86,8 @@ impress.jstp.mixin = (application) => {
     interfaceName // name of the interface to inspect
   ) => {
     const appInterface = application.api[interfaceName];
-    if (appInterface) {
-      return Object.keys(appInterface);
-    } else {
-      return null;
-    }
+    if (!appInterface) return null;
```

Comment:
Generally +1 for this change, but maybe even `return appInterface || Object.keys(appInterface);`? `null` is not required here, `undefined` is fine too.